hopr/hopr-lib/Cargo.toml

```diff
@@ -36,6 +36,8 @@ prometheus = [
   "hopr-strategy/prometheus",
 ]
 capture = ["hopr-transport/capture"]
+# disables filtering of private ip addresses in the p2p layer
+local-testing = ["hopr-transport/local-testing"]
```

Comment:
Interesting choice for this to be a feature instead of a runtime flag. 

So to run local cluster, we need to always build with this flag?

Reply:
That was the idea. But our flake setup takes care of this locally. We can also use a runtime flag, whichever is preferred.